blog/rome-incentives/README.md

```diff
@@ -0,0 +1,225 @@
+
+What you can do to qualify for getting paid on Rome is described below. More information, guides on how to get setup and extra support can be found in our helpdesk repo on GitHub.
+
+Paid Roles
+----------
+
+- Content Creator
+- Content Curator
+- Storage Provider
+- Validator
+- Council Member
+- Bug Reporter
+
+If you find one or more of these roles interesting, or want to learn about future roles on the network, we encourage you to visit the new [roles](https://joystream.org/roles) section on our website. Here you can browse through them and get a sense of what they will mean to the network, along with responsibilities and requirements.
+
+You can also sign up for role specific newsletters that will keep you updated on development and incentives!
+
+Content Creators
+----------------
+
+![](Content%20Creators.png)
+
+<i>Compete for $350 per week as a content creator</i>
+
+When the Joystream mainnet is live, `Content Creators` will play a fundamental role within the platform's ecosystem. This is because they will be responsible for creating and uploading the enormous variety of different content types and genres which we believe will allow Joystream to grow into a successful decentralized media platform.
+
+For the Rome testnet, we plan to pay a flat-rate Monero reward for every piece of content uploaded by `Content Creators` (with a different rate for each content type). We will also pay a $1.50 bonus on top of this rate for the 20 best pieces of content that are uploaded every week to encourage higher quality uploads. The maximum weekly incentive pool of $350 will be divided between these two categories of incentive.
+
+Prospective `Content Creators` should also be aware that we will only pay the item reward for content with complete metadata and which follows the set-out requirements for each content type (to be released at a later date).
+
+The `Content Curators` (see below) will be incentivized to find content with incorrect metadata and will likely remove it from the content directory. In these cases, content creators will not be paid. Multiple spam uploads which represent a burden to moderate for the `Content Curators` may even be penalized and result in deductions on payouts due for qualifying content uploads on your content creator profile.
+
+Uploading illegal or copyrighted content will result in a disqualification from payouts. We will provide `Content Creators` with a list of sources for safe and non-copyrighted material which can be uploaded as part of this testnet and will qualify for the payments discussed above and laid out in the table below.
+
+### Rewards based on content type
+
+| CONTENT TYPE | MAXIMUM PAYMENT PER ITEM | MAXIMUM WEEKLY POOL |
+| --- | --- | --- |
+| Short Video (Standard) | $0.35 per video* | $70 |
+| Long Video (Standard) | $0.50 per video* | $100 |
+| Album Item (Standard)** | $0.50 per track* | $100 |
```

Comment:
Music not yet available...

In general, I think this needs a little more thinking.